### ISSUE TITLE
TuyaOpenAPI Code28841101

### ISSUE THREAD

**Issue by Stesus** [tuya/tuya-homebridge#108]
I'm getting the TuyaOpenAPI error code 28841101 message "No permissions. This API is not subscribed". I also noticed the service "Smart Home Data Service" that is recommended to install doesn't exist on iot.tuya.com

**Comment by Stesus** [comment]
![image](https://user-images.githubusercontent.com/12788123/134078533-7ee624d2-6364-4fdc-895c-e34b82ea49b8.png)
I don't believe I am missing any APIs

**Comment by pparrilla** [comment]
I have this problem too

**Comment by weirdpattern** [comment]
I had this same issue, opened up a ticket in iot.tuya.com and they did something at the database level to unlock the account.
After this I was able to subscribe to all of these and as soon as that happened the authorizations started working.

Try that.

**Comment by Hanh94** [comment]
Hi guys, Sorry for the late reply.
We recently optimized APIs, Please see https://github.com/tuya/tuya-homebridge/wiki/Tuya-IoT-Platform-Configuration-Guide-Using-Smart-Home-PaaS?_source=d8fba44feeef4757f7f22a14c2295f3f#create-a-project

**Comment by starkprime** [comment]
I'm also having this issue. and can't find a a soluton

**Comment by Dunnlang** [comment]
Same. I just started trying to do Local Tuya today. Everything seems to be correct. I can't get the Device Information via IoT Core. I get the error:

"code": 28841106,
  "msg": "No permissions. This API is not subscribed.",

I appear to be properly subscribed to everything as nearly as I can tell. The APIs show as "In Service" and are authorized for my project.

**Comment by starkprime** [comment]
I verified those items as well. 
Authentication and API info seem correct to (I tried putting in a bogus number of password to try and get a different error) the issue doesn’t appear to be in my Auth setup. 
I even created a second project to try and reproduce. Same issue.

**Comment by Dunnlang** [comment]
I'm wondering if this needs to be manually enabled on a business day? That's what happens if you try to extend your service past the first month. I'm going to revisit this in a few days.

**Comment by drummingdemon** [comment]
Same here, enabled APIs and they say 'in service' - all the while not working in reality... please help! @Hanh94

**Comment by starkprime** [comment]
So my theory is that this is an account issue. 
I made a brand new tuya dev account and followed directions again. And it works. One thing I noticed that i do have in my new account vs what i have in my old account
I don't have have IOT-CORE service subscribed. Even though its shown as subscribed in another section.
![Screen Shot 2022-05-16 at 3 22 09 PM](https://user-images.githubusercontent.com/38358416/168666792-b43a6c7c-3d59-4513-8345-a84764edcb92.png)

![Screen Shot 2022-05-16 at 8 55 21 AM](https://user-images.githubusercontent.com/38358416/168666910-842222aa-19df-4431-97c9-47c71aedc3c9.png)
